src/plugins/muc-views/templates/occupant.js

```diff
@@ -29,6 +31,33 @@ const occupant_title = /** @param {MUCOccupant} o */(o) => {
     }
 }
 
+/**
+ * @param {MUCOccupant} o
+ */
+async function tplActionButtons (o) {
+    const buttons = await api.hook('getOccupantActionButtons', o, []);
+    if (!buttons?.length) { return '' }
+
+    const items = buttons.map(b => {
+        return html`
+        <button class="dropdown-item ${b.button_class}" @click=${b.handler}>
+            <converse-icon
+                class="${b.icon_class}"
+                color="var(--inverse-link-color)"
+                size="1em"
+            ></converse-icon>&nbsp;${b.i18n_text}
+        </button>`
+    });
+
+    if (!items.length) {
```

Comment:
This looks unnecessary because you already check `if (!button?.length)` on line 39.

Reply:
Fixed.